chainerx_cc/chainerx/stack_vector.h

```diff
@@ -194,4 +195,33 @@ class StackVector {
     BaseContainer d_{};
 };
 
+// Formatter to print StackVector containing integral elements as e.g. '[]' or '[1, 2, 3]'.
+template <typename T, stack_vector_detail::size_type N>
```

Comment:
There's no need of template. Just use `T=int64_t` and `N=kMaxNdim`.

Reply:
Will fix too.

Reply:
Fixed.